app/src/processing/app/Mode.java

```diff
@@ -565,77 +565,6 @@ public void actionPerformed(ActionEvent e) {
     }
   }
 
-
```

Comment:
See the section on comments at the end: https://github.com/processing/processing/wiki/Style-Guidelines